### PR TITLE
Feature/17

### PR DESCRIPTION
**Pull request by Remmo1**
Postgres db is on azure. I cleaned structure and add some dockerfiles. Also there is one big docker-compose

### CODE REVIEW

**Comment by Pandorianim** [comment]
Looks good

File: frontend/dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+COPY . .
+# Install dependencies (npm ci makes sure the exact versions in the lockfile gets installed)
+RUN npm install 
+# Build the app
+RUN npm run build
+
+# Expose port
+EXPOSE 3000
+# Start app
+CMD ["npm", "run", "dev"]
```
Comment (Veermove) [inline frontend/dockerfile:14]: Why is this service ran with dev profile in Dockerfile?

File: backend/database/README.md
Comment (Veermove) [inline backend/database/README.md:1]: This file in no longer actual. If you moved the docker-compose.yaml to root directory, please create there a README.md explaining how to run this docker-compose and what preconditions must be satisfied

**Comment by Veermove** [comment]
Oh, and you can still see secrets in git commits == they're still publicly available

File: backend/data-store/cmd/env/.env.template [hunk 2]
```diff
@@ -2,4 +2,4 @@ PG_STORE_USER=postgres
 PG_STORE_PASSWORD=postgres
 PG_STORE_DATABASE=datastore
 PG_STORE_HOST=localhost
-PG_STORE_PORT=5431
```
[the remark this answers is not in the document]
Comment (Remmo1) [reply backend/data-store/cmd/env/.env.template:5]: Doesn't matter, but I've never seen postgres on port 5431. It should run on 5432.

File: frontend/dockerfile (hunk 1 above)
Comment (Remmo1) [reply frontend/dockerfile:14]: from packet.json:
image
Profile dev is equivalent for next dev so it's an easy way to run nextjs app.

File: backend/database/README.md
Comment (Remmo1) [reply backend/database/README.md:1]: It's a good idea, one moment.

File: frontend/dockerfile (hunk 1 above)
Comment (gcfbn) [reply frontend/dockerfile:14]: `npm dev` will start a dev server, which is useful for local development. While running in container, `npm run build` and `npm run start` will do the job

**Review by Veermove — APPROVED** [review]

**Pull request merged**